### PR TITLE
Updated taskparam1 for tasktype 11

### PR DESCRIPTION
**Pull request by EmosewaMC**
Updated taskparam1 for tasktype 11 to have definitions for all possible parameters

### CODE REVIEW

File: docs/game-mechanics/mission-system.rst [hunk 1]
```diff
@@ -72,7 +72,14 @@ ObtainItem (11)
 The player needs to somehow obtain a count of :samp:`targetValue` items of the template specified in :samp:`target`.
 This is usually used to implement quests, asking the player to buy something from a vendor.
 
-Something is up with :samp:`taskParam1`.
+Depending on :samp:`taskParam1`:
+- 0 or no value: No extra parameters apply.
+- 1: The :samp:`targetLOT` item is not taken from the players inventory on mission turn in.
+- 2: The :samp:`targetLOT` item is taken from the players inventory on mission turn in.
+- 4: (UNUSED IN DATABSAE) If the :samp:`targetLOT` is removed from the players' inventory before mission completion,
```
Comment (Xiphoseer) [inline docs/game-mechanics/mission-system.rst:79]: Typo
```suggestion
- 4: (UNUSED IN DATABASE) If the :samp:`targetLOT` is removed from the players' inventory before mission completion,
```

**Comment by Xiphoseer** [comment]
See, the fact that 3 (i.e. `0b011`) is missing is what makes me think that this may be a bitmask i.e.

0b001 -> remove it
0b010 -> don't remove it
0b100 -> don't care for removal

and thus

5 = 0b101 -> don't care for removal but remove if it's still there.

But I'd merge this PR as is, unless we find clear evidence either way

**Comment by EmosewaMC** [comment]
> See, the fact that 3 (i.e. `0b011`) is missing is what makes me think that this may be a bitmask i.e.
> 
> 0b001 -> remove it 0b010 -> don't remove it 0b100 -> don't care for removal
> 
> and thus
> 
> 5 = 0b101 -> don't care for removal but remove if it's still there.
> 
> But I'd merge this PR as is, unless we find clear evidence either way

LCDR has mentioned it is a bitmask so I will update to reflect such!

**Comment by EmosewaMC** [comment]
Closing because I am making a new PR with more updates to this file

**Pull request closed without merging**